libs/interfaces/FileDetails.ts

```diff
@@ -1,6 +1,46 @@
 import { FileType } from "./FileType";
```

Comment:
New webhook events interface and needs to be updated. Missing upload pre & post related webhook event types.
https://docs.imagekit.io/api-reference/api-introduction/webhooks#list-of-events